ios/Internxt/AppDelegate.mm

```diff
@@ -0,0 +1,166 @@
+#import "AppDelegate.h"
```

Comment:
Please add a link reference to this source code or documentation if its own-made if possible

Reply:
this code is boilerplate only